docs/development.md

```diff
@@ -1,32 +1,141 @@
+      <p>Start the Docker daemon in another terminal, and leave it running in the background:</p>
+      <p><pre>$ dockerd</pre></p>
+    </td>
+    <td>
+      <p><pre># apt install docker</pre></p>
+      <p>Install <a href="https://docs.docker.com/docker-for-windows/install-windows-home">Docker Desktop</a>, which you'll be using as the Docker daemon, and leave it running in the background.</p>
+      <p>If you have set <code>appendWindowsPath=false</code> in your WSL config, then you may hit an error along the lines of <code>"docker-credential-desktop.exe": executable file not found in $PATH</code>. In this case you should either add <code>/mnt/c/Program\ Files/Docker/Docker/resources/bin</code> to your PATH, or <a href="https://github.com/rossjrw/dotfiles/blob/3c5445abb138b735cc3caf61f070c9125fa87d2f/.profile#L28">create a symlink</a> in a directory that is in your PATH already.</p>
+    </td>
+  </tr></tbody>
+</table>
+
+The first step is to build the Docker image, which is a single source of information for Docker to use in a container later:
+
+```
+$ cd installer
+$ docker build . -t scpwiki/wikijump:local --no-cache
+```
+
+Then load that image into a container, which is a VM that runs only the image you created:
+
+```
+$ docker run --name wj --publish 80:80 --publish 443:443 -it -d scpwiki/wikijump:local
+```
+
+This both creates a container for the image, and starts it. `docker run` is a shorthand for `docker create` and `docker start`.
 
-You can create a docker container using the following:
+Finally, navigate to https://www.wikijump.test in your browser. Your browser will probably complain that the site is insecure, citing that the page has a self-signed certificate. During local development, this is to be expected.
 
-```bash
-$ docker build . -t scpwiki/wikijump:local
-$ docker create --name wikijump -p 80:80 scpwiki/wikijump:local
-$ docker start wikijump
+-----
+
+To stop Wikijump:
+
+```
+$ docker stop wj
 ```
 
-Instead of building Wikijump locally, you can also pull the image from the Docker Hub:
+Note that this will only stop the process inside the container -- it won't destroy the container itself.
 
-```bash
-$ docker create --name wikijump -p 80:80 scpwiki/wikijump:latest
+It's useful to keep track of existing Docker images and containers, and destroy them when you no longer need them, so you don't waste space rebuilding the same image over and over. If you are using Docker Desktop, you can manage containers and images from the GUI. Otherwise, on command line:
+
+```
+$ docker container ls  # List containers
+$ docker rm [ID]       # Destroy the container with this ID
+$ docker images        # List images
+$ docker rmi [ID]      # Remove the image with this ID
 ```
 
-Then terminate it:
+### Installation via Docker from the Docker Hub
+
+Instead of building Wikijump locally, you can also pull the `latest` image from the Docker Hub, though bear in mind it may not be up to date the repository:
```

Comment:
```suggestion
Instead of building Wikijump locally, you can also pull the `latest` image from the Docker Hub, though bear in mind it may not be up to date with the repository:
```